File: src/BukkitPE/level/particle/EnchantParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;

class EnchantParticle extends GenericParticle{
	public function __construct(Vector3 $pos){
		parent::__construct($pos, Particle::TYPE_MOB_SPELL);
	}
}

File: src/main/java/net/BukkitPE/level/particle/EnchantParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class EnchantParticle extends GenericParticle {
    public EnchantParticle(Vector3 pos) {
        super(pos, Particle.TYPE_MOB_SPELL);
    }
}
